doc/rados/operations/pools.rst

```diff
@@ -737,8 +737,18 @@ Managing pools that are flagged with ``--bulk``
 ===============================================
 See :ref:`managing_bulk_flagged_pools`.
 
+.. _setting_values_for_a_stretch_pool:
+
 Setting values for a stretch pool
 =================================
+
+.. important:: Setting values for a stretch pool requires ``mon_osd_min_in_ratio <= 0.55``
+   to ensure that PGs get remapped to a new set of OSDs when the cluster
+   sustains multiple failures in different buckets. For example, a 3 site stretched cluster
```

Comment:
multiple failures across CRUSH trees

Buckets sort of connotes S3 which could be confusing.

Also s/3 site/three-site/